content/blog/2022/08/camel-karavan-3.18.1/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Karavan Preview Release 3.18"
+date: 2022-08-19
+authors: [mgubaidullin]
+categories: ["Releases","Tooling"]
+preview: "Karavan 3.18 - cloud-native integration toolkit"
+---
+
```

Comment:
For users that are not so familar with Karava can we maybe a few words first what Karavan is. IMHO its primary the UI designer and I would like to see this highlighted first.